sw/device/tock/boards/opentitan/tock_dependencies/Cargo_remote.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "tock_dependencies"
+version = "0.1.0"
+authors = ["Silvestrs Timofejevs <silvestrst@lowrisc.org>"]
+edition = "2018"
+
+[dependencies]
+components = { git = "https://github.com/tock/tock", rev = "d37ef3d2" }
+rv32i = { git = "https://github.com/tock/tock", rev = "d37ef3d2" }
+capsules = { git = "https://github.com/tock/tock", rev = "d37ef3d2"}
+kernel = { git = "https://github.com/tock/tock", rev = "d37ef3d2" }
+tock_rt0 = { git = "https://github.com/tock/tock", rev = "d37ef3d2" }
```

Comment:
I don't think we want to specify the revision here, it's handled in `Cargo.lock`. Eventually we will need CI tooling to advance this revision to upstream ToT, which I believe Cargo can mostly do for us if we use `Cargo.lock`.

Reply:
Yup I agree, I am not sure if I use "pre this change" Cargo.lock, whether it will keep the revision that it used to have, I can try it later.

Just to explain why I forced the revision here - it's just to make sure that we use the same revision as before this change for now. As the upstream kernel interface changed slightly, so latest tock version breaks our downstream board.